scripts/superdesk-authoring/authoring.js

```diff
@@ -2352,6 +2352,7 @@
                         })
                         .then(function(value) {
                             notify.success(gettext('Item sent.'));
+                            $rootScope.$broadcast('item:fetch');
```

Comment:
:+1: thanks for clarifying

On Friday, 15 July 2016, ioanpocol <notifications@github.com> wrote:

> In scripts/superdesk-authoring/authoring.js
> <https://github.com/superdesk/superdesk-client-core/pull/710#discussion_r70921257>
> :
>
> > @@ -2352,6 +2352,7 @@
> >                          })
> >                          .then(function(value) {
> >                              notify.success(gettext('Item sent.'));
> > +                            $rootScope.$broadcast('item:fetch');
>
> Fetch is an operation to add on archive an item from ingest list. The
> operation needs more changes compared with a simple move.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/superdesk/superdesk-client-core/pull/710/files/92e2a1269f4956c7e6d2a9a23b40eaf72ec61d17#r70921257>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AGYGlIIONUP--jsyc19MJQ35H-dfvQDtks5qVxE1gaJpZM4JKNiw>
> .
>